com.fightcade.Fightcade.Steamos.yml

```diff
@@ -0,0 +1,63 @@
+id: com.fightcade.Fightcade.Steamos
+runtime: com.fightcade.Fightcade
+runtime-version: stable
+sdk: org.freedesktop.Sdk//24.08
+build-extension: true
+separate-locales: false
+
+modules:
+  - name: libsecret
+    buildsystem: meson
+    build-options:
+      prefix: /app/steamos
+      lib: lib
+    config-opts:
+      - -Dmanpage=false
+      - -Dvapi=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+    cleanup:
+      - "/bin"
+      - "/include"
+      - "/lib/pkgconfig"
+      - "/share/gir-1.0"
+      - "/share/man"
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.6.tar.xz
+        sha256: 747b8c175be108c880d3adfb9c3537ea66e520e4ad2dccf5dce58003aeeca090 
+  - name: min
+    buildsystem: simple
+    build-commands:
+      # Replace the placeholder PREFIX with the actual destination path in the run script
+      - sed -i "s|PREFIX|${FLATPAK_DEST}|g" run.sh
```

Comment:
why do you need to `sed`?

Reply:
If `${FLATPAK_DEST}` is placed directly into the script section below, it results in this:

```
[📦 com.fightcade.Fightcade bin]$ cat min-browser 
#!/bin/sh
zypak-wrapper ${FLATPAK_DEST}/extra/min/min --no-gpu-sandbox --no-sandbox "$@" 2>&1
```

Which is not a valid variable at runtime.

The `sed` command allows us to insert the path dynamically (should things change/move around in the future), but if it's preferred we could hard code it.

```
[📦 com.fightcade.Fightcade ~]$ cat /app/steamos/bin/min-browser 
#!/bin/sh
zypak-wrapper /app/steamos/extra/min/min --no-gpu-sandbox --no-sandbox "$@" 2>&1
```

Reply:
because it's not available at runtime. Just put the actual value. It's better than sed. There not need to be complicated.

Reply:
Done